cms/envs/common.py

```diff
@@ -1115,3 +1120,25 @@
     'options': {},
 }
 PROCTORING_SETTINGS = {}
+
+
+############################ OAUTH2 Provider ###################################
+
+# OpenID Connect issuer ID. Normally the URL of the authentication endpoint.
+OAUTH_OIDC_ISSUER = 'https://www.example.com/oauth2'
```

Comment:
@jimabramson it looks like our configuration is already set up correctly, so we should get this for free. `OAUTH_OIDC_ISSUER` is defined as [`generic_env_config`](https://github.com/edx/configuration/blob/master/playbooks/roles/edxapp/defaults/main.yml#L739), so it should be included in `cms_env_config`.